crates/holochain/src/conductor/cell.rs

```diff
@@ -151,6 +154,7 @@ impl Cell {
                     holochain_p2p_cell,
                     queue_triggers,
                     init_mutex: Default::default(),
+                    enabled: AtomicBool::new(true),
```

Comment:
pretty sure the Cell will get reset to enabled every time it's recreated, which I believe would be on every restart.